controllers/articles.js

```diff
@@ -0,0 +1,247 @@
+import crypto from 'crypto';
+import models from '../models/index';
+
+const { article, User } = models;
+
+const articles = {
+
+  /*
+   * Deleting a post based to its slug.
+   *
+   * For directly publishing an article, status flag has to be passed
+   * into the request body object otherwise the article will be in drafted.
+   */
+  createAnArticle: async (req, res) => {
```

Comment:
Rename this and other similar variable/function names to `createArticle`, `updateArticle` etc